django_rq/workers.py

```diff
@@ -23,3 +23,20 @@ def get_worker(*queue_names):
     return Worker(queues,
                   connection=queues[0].connection,
                   exception_handlers=get_exception_handlers() or None)
+
+
+def collect_workers_per_configuration(queue_list):
```

Comment:
Ah, right. We can turn the keys into frozensets: https://stackoverflow.com/questions/13264511/typeerror-unhashable-type-dict but as you said it does look kludgy.

Let's keep it the way it is now then.